modules/nodes-pods-user-namespaces-configuring.adoc

```diff
@@ -6,36 +6,17 @@
 [id="nodes-pods-user-namespaces-configuring_{context}"]
 = Configuring Linux user namespace support
 
+You can configure your pods to run in Linux user namespaces by setting the `hostUsers` parameter to `false` in the workload spec, as shown in the following procedure. 
 
-.Prerequisites
+Running workloads in user namespaces allows you to specify `RunAsAny` for Security Context Constraint (SCC) fields, such as `fsGroup`, `runAsGroup`, `runAsUser`, and `supplementalGroups`. Setting these fields to `RunAsAny` can have specific benefits described throughout the {product-title} documentation. Within a user namespace, `RunAsAny` is generally considered to be a safe practice. For extra security, you can create a new SCC specifically for workloads in user namespaces by setting the `userNamespaceLevel: RequirePodLevel` field in the SCC, which requires that the workloads run in user namespaces. To require a specific SCC, set the `openshift.io/required-scc` annotation in any workload resource that can set a pod manifest template, such as a deployment or daemon set.
 
-* You enabled the required Technology Preview features for your cluster by editing the `FeatureGate` CR named `cluster`:
-+
-[source,terminal]
-----
-$ oc edit featuregate cluster
-----
-+
-.Example `FeatureGate` CR
-[source,yaml]
-----
-apiVersion: config.openshift.io/v1
-kind: FeatureGate
-metadata:
-  name: cluster
-spec:
-  featureSet: TechPreviewNoUpgrade <1>
-----
-<1> Enables the required `UserNamespacesSupport` and `ProcMountType` features.
-+
-[WARNING]
-====
-Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone and prevents minor version updates. This feature set allows you to enable these Technology Preview features on test clusters, where you can fully test them. Do not enable this feature set on production clusters.
-====
-+
-After you save the changes, new machine configs are created, the machine config pools are updated, and scheduling on each node is disabled while the change is being applied.
+Also, configuring workloads as `procMount: unmasked` is generally considered as safe within a user namespace. Setting `procMount` to `unmasked` has benefits that are beyond the scope of this documentation. 
+
+To ensure that the namespace functionality exists on all nodes that you want to run in a user namespace, you can configure the minimum version of kubelet that is required for the nodes in your cluster. If the kubelet version in your cluster is lower than this version, new nodes are not scheduled and existing nodes are marked as degraded. For existing nodes with a lower version, the kubelet can only read the node object by using `oc get` or `oc update` commands, and the actions that the node can perform, by using a `SelfSubjectAccessReview`. The node is not allowed to gain access to any other API objects.     
```

Comment:
I think it would be worth really hammering out why an admin would do this: kubelets before 1.30.0 do not fail to create a pod if a user namespace can't be created. thus, the apiserver shouldn't trust that a kubelet will and relax validation. an admin saying "minKubeletVersion: 1.30.0" is a way of proving to themselves and the apiserver that there won't be nodes that will say a pod was created with a user ns when it wasn't